crates/resolver/src/config.rs

```diff
@@ -607,9 +607,16 @@ fn default_recursion_desired() -> bool {
 }
 
 /// The lookup ip strategy
+#[non_exhaustive]
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum LookupIpStrategy {
+    /// Query for AAAA and A in parallel and prefer the AAAA result
```

Comment:
Should we add documentation on the Ipv4AndIpv6 strategy that mentions people might instead want to use HappyEyeballs? The distinction being the delay to see if the AAAA comes back before returning.